Sources/Amplitude/Amplitude.m

```diff
@@ -279,6 +286,12 @@ - (instancetype)initWithInstanceName:(NSString *)instanceName {
             if (previousSessionId >= 0) {
                 self->_sessionId = previousSessionId;
             }
+            
```

Comment:
Why we still use SQLite to store SEQUENCE_NUMBER instead of  NSUserDefult?

Reply:
Accidentally kept some of the old logic. I thought this would be handled through your migration code (this is the initialization stage of the SDK and the DB helper)

Reply:
+1